### PR TITLE
feat: add contact to job detail page

### LINKED ISSUE #393: Display contact person for job
As suggested by Katja in Slack.

Needs some discussion. I would suggest the following:

- Display area responsible by default
- Have option to override contact in job type and in job instance

Advantage compared to just putting a contact in the description is, that there would always be some contact by default that should at least know whom to forward the request to.

### PR DESCRIPTION
**Pull request by zesinho**
Adds contact info to job detail page.

Fixes #393 

Also replaced line-breaks for styling with padding-bottom.

### CODE REVIEW

File: juntagrico/templates/job.html [hunk 1]
```diff
@@ -15,13 +15,12 @@ <h3>
 {% endblock %}
 {% block content %}
     {% vocabulary "assignment_pl" as v_assignment_pl %}
-    <div class="row">
+    <div class="row pb-4">
```
Comment (motu55) [inline juntagrico/templates/job.html:18]: Please don not just ad spacing as you see fit, the look has to be consinsten over the whole system

File: juntagrico/templates/job.html [hunk 2]
```diff
@@ -130,11 +122,21 @@ <h3>
...
         </div>
     {% endif %}
-    <div class="row">
+    <div class="row pb-4">
+        <div class="col-md-3">
+            {% trans "Kontakt" %}:
+        </div>
+        <div class="col-md-5">
+                {{ job.type.activityarea.coordinator|richtext|safe }}
+                <a class="ml-1" href="{% url 'contact-member' member_id=job.type.activityarea.coordinator.pk %}">
```
Comment (motu55) [inline juntagrico/templates/job.html:134]: this is  a mailto link in order to be consistent with the rest of the system

**Review by motu55 — CHANGES_REQUESTED** [review]

File: juntagrico/templates/job.html (hunk 1 above)
Comment (pascalfree) [reply juntagrico/templates/job.html:18]: this spacing replaces a `<br>` below, so it should not change how it looks.
In some cases, which I commented below, the br was right and should be changed back. But in general I think this is the way to go.
Maybe it should be part of a separate pull request and we do that on all templates.

File: juntagrico/templates/job.html (hunk 2 above)
Comment (pascalfree) [reply juntagrico/templates/job.html:134]: In the related issue you write that the envelope should be used if I'm not mistaken.

File: juntagrico/templates/job.html [hunk 3]
```diff
@@ -15,13 +15,12 @@ <h3>
...
     {% vocabulary "assignment_pl" as v_assignment_pl %}
-    <div class="row">
+    <div class="row pb-4">
         <div class="col-md-3">
             {% trans "Zeitpunkt" %}:
         </div>
         <div class="col-md-5">
-            {{ job.time |date:"l, d.m.Y, H:i" }} - {{ job.end_time|date:"H:i" }}
-            <br/>
+            <div class="pb-4">{{ job.time |date:"l, d.m.Y, H:i" }} - {{ job.end_time|date:"H:i" }}</div>
```
Comment (pascalfree) [inline juntagrico/templates/job.html:23]: I think here the br was legit, as it was not used to add spacing but to break this line.

File: juntagrico/templates/job.html [hunk 4]
```diff
@@ -197,19 +198,16 @@ <h3>
                                 {% endfor %}
                             </select>
                         </div>
-                        <br/>
                     {% else %}
                         <input type="hidden" name="jobs" value="1"/>
                     {% endif %}
                     {% for extra in job.empty_per_job_extras %}
-                        <input type="checkbox" name="extra{{ extra.extra_type.id }}" value="{{ extra.extra_type.id }}"/>
+                        <input class="pb-4" type="checkbox" name="extra{{ extra.extra_type.id }}" value="{{ extra.extra_type.id }}"/>
```
Comment (pascalfree) [inline juntagrico/templates/job.html:205]: Here as well the br is legit. now it adds space, that wasn't there before.

File: juntagrico/templates/job.html [hunk 5]
```diff
@@ -197,19 +198,16 @@ <h3>
...
                         {{ extra.extra_type.name }}
-                        <br/>
                     {% endfor %}
                     {% for extra in job.per_member_extras %}
-                        <input type="checkbox" name="extra{{ extra.extra_type.id }}" value="{{ extra.extra_type.id }}"/>
+                        <input class="pb-4" type="checkbox" name="extra{{ extra.extra_type.id }}" value="{{ extra.extra_type.id }}"/>
```
Comment (pascalfree) [inline juntagrico/templates/job.html:209]: Here as well.

**Review by pascalfree — CHANGES_REQUESTED** [review]

File: juntagrico/templates/job.html (hunk 2 above)
Comment (zesinho) [reply juntagrico/templates/job.html:134]: yes i understand the same:

"so for the moment i would say display the area main coordinator and also the envelope icon to send an email use the existing simple contact form (and sendign mechanism which is used to send a message to the info email address"

File: juntagrico/templates/job.html (hunk 1 above)
Comment (zesinho) [reply juntagrico/templates/job.html:18]: i reverted the ones adding spaces which wasn't there before

File: juntagrico/templates/job.html (hunk 3 above)
Comment (zesinho) [reply juntagrico/templates/job.html:23]: reverted

File: juntagrico/templates/job.html (hunk 4 above)
Comment (zesinho) [reply juntagrico/templates/job.html:205]: reverted

File: juntagrico/templates/job.html (hunk 5 above)
Comment (zesinho) [reply juntagrico/templates/job.html:209]: reverted

**Comment by zesinho** [comment]
reverted the commented legit line-breaks back. 

let me know if you want the line-breaks in a separate PR.

**Review by pascalfree — COMMENTED** [review]
For me the PR is good. However I would wait with merging and put the contact information depending on the discussion on #393.

**Comment by pascalfree** [comment]
moving this to 1.5 as it now requires migrations

**Pull request merged**